src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -130,7 +139,8 @@ export class SearchMenu extends React.Component {
             <button
               type="submit"
               disabled={!validUserInput}
-              className="vads-u-margin-left--0p25"
+              className="vads-u-margin-left--0p25 vads-u-margin-right--0p5 "
+              onSubmit={() => this.handleSearchEvent()}
```

Comment:
I was going to suggest making this `onSubmit={this.handleSearchEvent}` but now I see that it usually gets passed a suggestion and onSubmit would pass it an event, so this makes sense! 👍

Reply:
Huh, TIL that `onSubmit` works when attached to a button. I thought it had to be attached to a `<form>` element.

Reply:
Just double checking but will this event will still be called even when the user submits the form by pressing the enter key?